packages/patternfly-4/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -240,7 +241,7 @@ export const SideNavLayout = ({
   return (
     <div>
       <Helmet>
-        <title>{title}</title>
+      <title>{title}{pageTitle ? `- ${pageTitle}` : ``}</title>
```

Comment:
Nit:
```js
pageTitle && `- ${pageTitle}`
```

Reply:
Another nit: not sure why, but intuition is telling me that either a space before the dash or a colon instead of the dash would look better.
"PatternFly 4- About" 🤔 
"PatternFly 4: About" ✅ 
"PatternFly 4 - About" ✅

Reply:
Thanks - good suggestions!